axelor-base/src/main/java/com/axelor/apps/base/service/ImportExportTranslationServiceImpl.java

```diff
@@ -0,0 +1,357 @@
+                    + "where meta_translation.language = ANY ( "
+                    + "select base_language.code "
+                    + "from base_import_export_translation "
+                    + "left join base_import_export_translation_language_set "
+                    + "on base_import_export_translation_language_set.base_import_export_translation = base_import_export_translation.id "
+                    + "left join base_language "
+                    + "on base_import_export_translation_language_set.language_set = base_language.id "
+                    + "where base_import_export_translation.id = :curId ) "
+                    + "limit :limit "
+                    + "offset :offset ")
+            .setParameter("curId", importExportTranslation.getId())
+            .setParameter("limit", LIMIT)
+            .setParameter("offset", offset);
+    return messageQuery.getResultList();
+  }
+
+  /**
+   * This method find the location in headers[] of a specific languageCode. Note that this headers
+   * array is for the csv file headers.
+   *
+   * @param languageCode
+   * @param headers
+   * @return The index of that languageCode in headers[]
+   */
+  protected int findPosition(String languageCode, String[] headers) {
+    for (int i = 1; i < headers.length; i++) {
+      if (languageCode.equals(headers[i])) {
+        return i;
+      }
+    }
+    return 0;
+  }
+
+  /**
+   * This method queries the languageCode corresponding to the id of the importExportTranslation
+   * object.
+   *
+   * @param importExportTranslationId
+   * @return A List&lt;String&gt; type list containing all the language codes corresponding to the
+   *     importExportTranslation object.
+   */
+  protected List<String> getLanguageHeaders(long importExportTranslationId) {
+    Query languageQuery =
+        JPA.em()
+            .createNativeQuery(
+                "select base_language.code "
+                    + "from base_import_export_translation "
+                    + "left join base_import_export_translation_language_set "
+                    + "on base_import_export_translation_language_set.base_import_export_translation = base_import_export_translation.id "
+                    + "left join base_language "
+                    + "on base_import_export_translation_language_set.language_set = base_language.id "
+                    + "where base_import_export_translation.id = :curId ")
+            .setParameter("curId", importExportTranslationId);
+    return (List<String>) languageQuery.getResultList();
+  }
+
+  /**
+   * This method imports the translation file and saves each record into ImportExportTranslation
+   * obj.
+   *
+   * @param importExportTranslation
+   * @return
+   */
+  @Transactional
+  @Override
+  public Path importTranslations(ImportExportTranslation importExportTranslation) {
+    errorNumber = 0;
+    Path filePath = MetaFiles.getPath(importExportTranslation.getUploadFile());
+    char separator = ';';
```

Comment:
create a global variable private static final and use it in the export method too